EIPS/eip-3455.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 3455
+title: SUDO Opcode
+author: William Morriss (@wjmelements), Baptiste Vauthey (@thabaptiser)
+discussions-to: https://ethereum-magicians.org/t/eip-3455-sudo-opcode/5860
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-04-01
+---
+
+## Simple Summary
+A new opcode is introduced to allow calling from an arbitrary sender address.
+
+## Abstract
+A new opcode, `SUDO`, is introduced with the same parameters as `CALL`, plus another parameter to specify the sender address.
```

Comment:
I really like this proposal and I'm sure there are no security issues. Wouldn't it be even more flexible to allow the `SUDO` opcode to perform `DELEGATECALL` on any sender's behalf though?